data/saas/config/yotpo_loyalty_config.yml

```diff
@@ -14,10 +14,12 @@ saas_config:
       label: GUID
     - name: api_key
       label: API Key
+      sensitive: True
```

Comment:
What is the API key field used for? As far as I can tell, Yotpo auth only needs the store ID & secret key?

https://core-api.yotpo.com/reference/authentication

Reply:
The Yotpo Loyalty API doesn't allow programatic erasures through any of the documented API endpoints. We bypassed this by inspecting the network traffic when a user is deleted from the Yotpo Loyalty dashboard and reproducing the call in the connector config. This endpoint requires the API key that is found at https://loyalty-app.yotpo.com/general-settings. Might not be the best approach but it works 😬.